### PR TITLE
[MM-61506] Fix errcheck issues in server/channels/app/teams/helper_test.go

### PR DESCRIPTION
**Pull request by Gesare5**
#### Summary

Fixes errcheck issue in `server/channels/app/teams/helper_test.go`

#### Ticket Link

Fixes  [#29110](https://github.com/mattermost/mattermost/issues/29110)
Jira  [https://mattermost.atlassian.net/browse/MM-61506](https://mattermost.atlassian.net/browse/MM-61506)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by hanzei — APPROVED** [review]
The changes are fine. :+1: 

If you want to go a step further, we can use this PR as an opportunity to cleanup the use of `panic` in the file. You can use https://github.com/mattermost/mattermost/pull/28936 as an example.

**Comment by Gesare5** [comment]
Okay, let me take a look at that @hanzei

File: server/channels/app/teams/helper_test.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/mattermost/mattermost/server/public/shared/request"
 	"github.com/mattermost/mattermost/server/v8/channels/store"
 	"github.com/mattermost/mattermost/server/v8/config"
+	"github.com/stretchr/testify/require"
```
Comment (hanzei) [inline server/channels/app/teams/helper_test.go:17]: Nit: Please move this import into a separate group before the `github.com/mattermost/mattermost/server`  ones

**Review by hanzei — APPROVED** [review]
Awesome :tada:

**Review by larkox — APPROVED** [review]

**Pull request merged**